x/gamm/types/account_test.go

```diff
@@ -381,7 +381,7 @@ func TestPoolAccountMarshalYAML(t *testing.T) {
 	require.NoError(t, err)
 
 	want := `|
-  address: cosmos10vgzkwmgvqs5nwh3mprrt53ff6r7autptppdxh
+  address: cosmos1m48tfmd0e6yqgfhraxl9ddt7lygpsnsrhtwpas
```

Comment:
What is this for?

Reply:
This is the module address, but it has changed due to the store keys changing